client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -160,7 +174,13 @@ class CustomContentTypes extends Component {
 				'you can display them using the shortcode [portfolio].',
 			{
 				components: {
-					link: <a href={ localizeUrl( 'https://wordpress.com/support/portfolios/' ) } />,
+					link: (
+						<a
+							onClick={ this.onSupportLinkClick }
+							href={ localizeUrl( 'https://wordpress.com/support/portfolios/' ) }
+							post-id={ 84808 }
+						/>
```

Comment:
Was https://wpcalypso.wordpress.com/devdocs/design/inline-support-link not a good fit here?

Reply:
Oop. You totally did also link that in our Slack conversation, I now see; I followed your link to https://github.com/Automattic/wp-calypso/pull/40937 and got sidetracked by the pattern offered in https://github.com/Automattic/wp-calypso/blob/master/client/blocks/support-article-dialog/README.md. `InlineSupportLink` looks cleaner, though, so I'll refactor to that. 😄